src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -401,7 +409,13 @@ export default {
 				console.error(error)
 				showError(t('spreed', 'Could not update the message'))
 			}
-		}
+		},
+
+		setIsEditing({ messageId, value }) {
+			if (messageId === this.message.id) {
+				this.isEditing = value
+			}
+		},
```

Comment:
Maybe use store  to store the id that is being processed instead and put computed in this component here ?

Reply:
There could be several 'processing' messages at the time, so store soultion could get complicated. I think, it's easier to send a signal twice.

Reply:
fiine